Readme.md

```diff
@@ -265,6 +272,43 @@ Example:
  Console.WriteLine(result);
 ```
 
+### Working with non-normalized fractions
+
+For performance reasons, as of version 8.0.0, mathematical operations no longer automatically generate normalized fractions if one of the operands is an improper (i.e. non-normalized) fraction. This has an impact on your calculations, especially if you have used the `JsonFractionConverter` with default settings. In such cases, deserialized fractions create improper fractions, which can lead to changed behavior when calling `Equals` and `ToString`.
+
+| Symbol | Description                                                                                          |
+| ------ | ---------------------------------------------------------------------------------------------------- |
+| $NF$   | Non-normalized (possibly improper) fraction, created with `normalize: false`                         |
+| $F$    | Fraction created with `normalize: true`                                                              |
+| $⊙$   | Mathematical operation having two operands ($+$, $-$, $*$, $/$, $mod$).                              |
+
+The following rules apply:
+
+$F ⊙ F = F$
+$F ⊙ NF = NF$
+$NF ⊙ F = NF$
+$NF ⊙ NF = NF$
+
+That said, the following applies for normalized fractions:
+
+```csharp
+var a = new Fraction(1, 2, normalize: true);
+var b = new Fraction(2);    // is 2/1 (automatically normalized)
+var result = a * b;         // result is 1/1
+```
+
+$\frac{1}{2}*\frac{2}{1}=\frac{1}{1}$
+
+However, for non-normalized fractions the following applies:
+
+```csharp
+var a = new Fraction(1, 2, normalize: false);
+var b = new Fraction(2);    // is 2/1 (automatically normalized)
+var result = a * b;         // result is 2/2

```

Comment:
The result here would actually be 1/1 with `FractionState` = `Unknown`

Reply:
Right, I forgot the `reduceTerms` in `Fraction.Multiply(..)`. Corrected, thanks.